assets/js/steps/CheckInItems.jsx

```diff
@@ -160,22 +160,24 @@ function CheckInItems({ actionHandler }) {
     /**
      * Handle successful checkin.
      */
-    let booksLength = items.filter(
+    const newCheckInLength = items.filter(
       (book) =>
         book.status === BookStatus.CHECKED_IN && book.message !== "Reserveret"
     ).length;
-    if (booksLength > checkedInBooksLength) {
-      setCheckedInBooksLength(booksLength);
+
+    if (newCheckInLength !== checkedInBooksLength) {
+      setCheckedInBooksLength(newCheckInLength);
     }
 
     /**
-     * Play sound for check-in error.
+     * Play sound for check-in error or if the item should be delivered to another library.
      */
-    booksLength = items.filter(
-      ({ status }) => status === BookStatus.ERROR
+    const newErrorsLength = items.filter(
+      ({ status, message }) => status === BookStatus.ERROR || message?.indexOf("Sendes til") === 0
```

Comment:
1. Is this best we can do? I'm guessing I know the answer, but sh*t it's ugly having to parse the message like this.
2. `Sendes til` is hardcoded and used in two different files. Should be a constant defined in `constants.js`